### PR TITLE
FOIA-407: Removed Undefined.

### PR DESCRIPTION
**Pull request by sundarcreosen**
Please verify @brockfanning

### CODE REVIEW

**Comment by sundarcreosen** [comment]
@brockfanning Please verify.

**Comment by brockfanning** [comment]
@sundarcreosen This removes the "Undefined", and keeps the "This field has a maximum length..." message., but it loses whatever was already set as help text in the webform. The intention of that code was to *add* the message "This field has a maximum length..." to whatever help text was already there (if any).

For example, if the help text in the webform was `<p>Please enter a description of your request.</p>`, then after this code runs, it should be something like:

```
<p>Please enter a description of your request.</p>
<em>This field has a maximum length...</em>`.
```

In other words, this hardcoded help text needs to be *appended to* the existing webform help text.
